src/test/java/uk/gov/hmcts/reform/civil/utils/HearingUtilsTest.java

```diff
@@ -60,4 +61,22 @@ void shouldReturnFee6_whenGivenAnyClaimFee() {
     void shouldReturnFee7_whenGivenAnyClaimFee() {
         assertThat(HearingUtils.getFastTrackFee(500000)).isEqualTo(new BigDecimal(34600));
     }
+
+    @Test
+    void shouldReturnFormattedFee_whenGivenAnyClaimFee() {
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(new BigDecimal(34600)).build())).isEqualTo("£346");
+    }
+
+    @Test
+    void shouldReturnFormattedFee2_whenGivenAnyClaimFee() {
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(new BigDecimal(132000)).build())).isEqualTo("£1,320");
+    }
+
+    @Test
+    void shouldReturnFormattedFee3_whenGivenAnyClaimFee() {
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(new BigDecimal(5000)).build())).isEqualTo("£50");
+    }
```

Comment:
These three can be turned into one `@ParameterizedTest`. Please see if you can do it.

Also, if the hearing fee allows for pences, please add relevant test parameters to test also non-whole values.

Reply:
Done